data-prepper-core/src/test/java/org/opensearch/dataprepper/peerforwarder/PeerForwardingProcessingDecoratorTest.java

```diff
@@ -28,13 +28,16 @@ class PeerForwardingProcessingDecoratorTest {
     @Mock
     Processor processorMock;
 
+    @Mock
+    PeerForwarder peerForwarder;
```

Comment:
We should be asserting that this is called with the correct values.

Also, we should verify the decorator returns what was provided by this mock.